src/main/java/com/recipapp/model/Sequence.java

```diff
@@ -0,0 +1,20 @@
+package com.recipapp.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+@Data
+@AllArgsConstructor
+public class Sequence {
+
+    @Id
+    private Integer id;
+    private Long seq;
```

Comment:
What do you think of changing this name to lastUsedIdInSequence or something similar?, maybe we can find a more representative name to this

Reply:
This is only for the document, who has no idea how anyone is going to use it. The actual purpose is given by the service, his only objective in life is to save a sequence